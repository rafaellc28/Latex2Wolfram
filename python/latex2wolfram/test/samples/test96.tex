% Test gradient expression
\nabla(\sin(x^{2} \cdot y))
