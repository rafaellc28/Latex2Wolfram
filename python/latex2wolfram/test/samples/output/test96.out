grad(sin(x ^ 2 * y))
